homeassistant/helpers/automation.py

```diff
@@ -0,0 +1,49 @@
+"""Helpers to configure automation-like objects."""
+
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_ACTION,
+    CONF_ACTIONS,
+    CONF_CONDITION,
+    CONF_CONDITIONS,
+    CONF_TRIGGER,
+    CONF_TRIGGERS,
+)
+
+
+def backward_compatibility_schema(value: Any | None) -> Any:
```

Comment:
I think this should be in `config_validation.py`

Reply:
Proposal for new feature in `config_validation.py`: https://github.com/home-assistant/core/pull/143834